.github/workflows/cuda.yml

```diff
@@ -1,6 +1,9 @@
 name: 🐧 CUDA
 
-on: [push, pull_request]
+on:
+  push:
+  pull_request:
+    types: [review_requested, ready_for_review]
```

Comment:
Upsi, should not have been changed.
Fixed in 3dc6b72d4650427afbb95b350ecceaa47f7d4546